Micronaut Cache Hazelcast guide (#658)
micronaut-projects/micronaut-guides: guides/micronaut-cache/micronaut-cache.adoc

=== Micronaut Cache API

> Imagine a service which retrieves headlines for a given month. Depending on the exact retrieval operation (e.g, accessing a database or making an HTTP call), it may be expensive and you would want to cache it.

Reviewer: ```suggestion
Imagine a service that retrieves headlines for a given month. Depending on the exact retrieval operation (e.g., accessing a database or making an HTTP call), this operation may be expensive, so you would want to cache it.
```

Rule of thumb: If you can't put a comma in front of it, "which" should be changed to "that." 

i.e. and e.g. always have a period *and* a comma (which is weird looking, but that's the rule. ¯\_(ツ)_/¯)
